### ISSUE TITLE
importlib.machinery.SourceFileLoader: False E1120: No value for argument 'fullname' in method call

### ISSUE THREAD

**Issue by bdrung** [PyCQA/pylint#6431]
### Bug description

```python
"""example.py"""
# pylint: disable=deprecated-method

import importlib

example = importlib.machinery.SourceFileLoader("example", "path/to/example.py")
example.load_module()
```

### Command used

```shell
pylint example.py
```

### Pylint output

```shell
************* Module example
example.py:7:0: E1120: No value for argument 'fullname' in method call (no-value-for-parameter)
```

### Expected behavior

I expect no errors.

### Pylint version

```shell
pylint 2.13.7
astroid 2.11.3
Python 3.10.4 (main, Apr  2 2022, 09:04:19) [GCC 11.2.0]
```

### OS / Environment

Ubuntu 22.04

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue. According to the [documentation](https://docs.python.org/3/library/importlib.html#importlib.machinery.SourceFileLoader) it's a false positive

**Comment by jacobtylerwalls** [comment]
The documentation doesn't match the signature to be friendlier to "manual calls" that see a fullname injected anyway, see [bpo-21438](https://bugs.python.org/issue21438). The method is deprecated; I suggest wontfix.
